pulp_2to3_migration/tests/functional/test_rpm_rerun.py

```diff
@@ -343,6 +349,21 @@ def test_rpm_distributors_only_added_or_with_changed_config(self):
 
         self.assertEqual(new_distribution_count, self.repo_info['new_distributions'])
 
+    def test_old_premigrated_content_removed(self):
+        """
+        Test that premigrated content is removed if the corresponding Pulp 2 content is removed.
```

Comment:
test that premigrated orphaned content

Reply:
not sure whether it worth pointing this out but at least it would be accurate based on the snapshots

Reply:
There is no such thing as orphaned premigrated content. Premigrated content is in Pulp2Content, it can't be orphaned, imo. Or we are introducing new terms :)
I initially wrote this comment with the word `orphaned` somewhere and it was very confusing, so I just wrote that the Pulp 2 content is removed.

Reply:
orphaned in sense that when it was premigrated it did not belong to any of the pulp2 repos

Reply:
It might have belonged to some, maybe it became orphaned and was removed in between on migration runs.
The key is that it was pre-migrated but not migrated. And on the next run it dissappeared from pulp2 completely.